config/sidekiq.yml

```diff
@@ -4,7 +4,7 @@
 # pick it up automatically.
 ---
 :verbose: false
-:concurrency: 10
```

Comment:
any specific reasons why we are making this change ?

Reply:
Yes, because sidekiq in docker- compose wasnt running as the rails app used up most of the redis connection pool, so i needed to adjust the concurrency so that both rails ans sidekiq runs fine. I also faces the concurrency issue with heroku with a free redis tier.

Reply:
Can we increase the connection pool on the Redis used in docker-compose? In Heroku free tier for Redis, the connection limit is 20 as mentioned here https://elements.heroku.com/addons/heroku-redis.